pkgs/build-support/build-dotnet-module/hooks/dotnet-fixup-hook.sh

```diff
@@ -0,0 +1,42 @@
+declare -a makeWrapperArgs gappsWrapperArgs
+
+# First argument is the executable you want to wrap,
+# the second is the destination for the wrapper.
+wrapDotnetProgram() {
+    makeWrapper "$1" "$2" \
+        --set "DOTNET_ROOT" "@dotnetRuntime@" \
+        --suffix "LD_LIBRARY_PATH" : "@runtimeDeps@" \
+        "${gappsWrapperArgs[@]}" \
+        "${makeWrapperArgs[@]}"
+
+    echo "Installed wrapper to: "$2""
+}
+
+dotnetFixupHook() {
+    echo "Executing dotnetFixupPhase"
+
+    if [ ! -z "${executables-}" ]; then
+        for executable in $executables; do
+            execPath="$out/lib/${pname}/$executable"
```

Comment:
I think there should be a way to override this if pname does not match the lib name.

Reply:
Good idea, that could be useful in scenarios where `$out/lib/$pname` is already used as well. I've added the `installPath` flag :+1:

Reply:
So, um, `installPath` was added as a flag to `buildDotnetModule`, but as I learned today, it doesn't seem to be used anywhere or do anything 😅

I imagine it probably got lost during refactoring, but I've opened #251267 about it